system-variables.md

```diff
@@ -710,6 +710,31 @@ For a system upgraded to v5.0.0-rc from an earlier version, if you have not modi
 - Default value: 60
 - This variable is used to set the step of the Prometheus statement generated when querying `METRIC_SCHEMA`. The unit is second.
 
+## tidb_multi_statement_mode <span class="version-mark">New in v4.0.11</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: OFF
+- Permitted values: OFF, ON, WARN
+- To reduce the impact from SQL injection attacks, TiDB now prevents multiple queries being executed in the same `COM_QUERY` call by default.
+- This variable is intended to be used as part of an upgrade path from earlier versions of TiDB. The following behaviors apply:
+
+| Client setting         | `tidb_multi_statement_mode` value | Multiple statements permitted? |
+|------------------------|-----------------------------------|--------------------------------|
+| Multi Statements = ON  | OFF                               | Yes                            |
+| Multi Statements = ON  | ON                                | Yes                            |
+| Multi Statements = ON  | WARN                              | Yes                            |
+| Multi Statements = OFF | OFF                               | No                             |
+| Multi Statements = OFF | ON                                | Yes                            |
+| Multi Statements = OFF | WARN                              | Yes (+warning returned)        |
+
+> **Note:**
+>
+> Only the default value of `OFF` can be considered safe. Setting `tidb_multi_statement_mode=ON` might be required if your application was specifically designed for an earlier version of TiDB. If your application requires multiple statement support, it is recommended to use the setting provided by your client library instead of the `tidb_multi_statement_mode` option. For example:
```

Comment:
Why "If your application **was** specifically designed for an earlier version" and "If your application **requires**" are in different tenses?
```suggestion
> Only the default value of `OFF` can be considered safe. If your application was specifically designed for an earlier version of TiDB, you might need to set `tidb_multi_statement_mode=ON`. If your application requires the "multiple statements" support, it is recommended to use the setting provided by your client library instead of the `tidb_multi_statement_mode` option. Some of your client libraries might be as follows:
```

Reply:
This reads kind of awkward because it starts two sentences with "If your application". Because "multiple statements" in this context is a concept, I don't think we should put it in quotes.

Can we leave as is?